source/sass/utilities.scss

```diff
@@ -7,10 +7,6 @@
   flex: 1;
 }
 
-.position-relative {
```

Comment:
Bootstrap has this already: https://github.com/twbs/bootstrap/blob/v4.6.0/scss/utilities/_position.scss#L4-L6

Reply:
❗